frontend/src/v5/ui/controls/formSelect/formSelect.component.tsx

```diff
@@ -14,12 +14,13 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { FormControl, SelectProps } from '@mui/material';
+import { FormControl, SelectProps, InputLabel } from '@mui/material';
 import { Controller } from 'react-hook-form';
-import { Select, InputLabel } from './formSelect.styles';
+import { Select } from './formSelect.styles';
 
 export type FormSelectProps = SelectProps & {
 	control: any;
+	name: string;
```

Comment:
This is not necessary the name is already part of SelectProps unless I guess you want to make it not optional. If we didnt have problems with forgetting that dont change it.

Reply:
Indeed it is because of that. In the creation of the form multiselect component, I forgot to pass the name and I was getting a non-exhaustive error. 
![image](https://user-images.githubusercontent.com/43547703/192505268-9f316739-324e-4c7e-84b4-3fb8a32695d8.png)
Since the name is required by the controller that wraps up the MUI select component, name is no longer optional. 
But I can revert this if you want